locales/en/index.yml

```diff
@@ -230,6 +230,8 @@ wallet:
     newMethod: Choose another payment method
     addDescription: You can add your payment methods (e.g. credit cards, bank accounts)
       so that you don't have to re-add them from scratch for every transaction
+    successful: The wallet was added successfully!
```

Comment:
I think _wallet_ is a term we use internally, we should probably use _payment method_ or just _credit card_ since we only support that one.